web-console/src/views/query-view/query-output/column-rename-input/column-rename-input.tsx

```diff
@@ -44,12 +44,12 @@ export const ColumnRenameInput = React.memo(function ColumnRenameInput(
       value={newName}
       onChange={(e: any) => setNewName(e.target.value)}
       onKeyDown={(e: React.KeyboardEvent<HTMLInputElement>) => {
-        switch (e.keyCode) {
-          case 13: // Enter
+        switch (e.key) {
+          case 'enter': // Enter
```

Comment:
Ditto on the capitalization here and everywhere

Reply:
Also now we do not need the comment. Yay for self documenting code.

Reply:
Done! Changed to 'Enter' and 'Tab'. Also, removed unwanted comments.

For testing and reference https://css-tricks.com/snippets/javascript/javascript-keycodes/